### DISCUSSION TITLE
Entered Shelter Date - can this be edited?

### DISCUSSION THREAD

**Discussion by kernny** [sheltermanager/asm3#1373] [Q&A / Help] [answered]
If a user puts the wrong date in the Entered Shelter Date field, can an admin update this field or does the entry have to be deleted and started again?

Thanks in advance!!!

**Comment by Tetzor** [comment, marked as answer]
Hi, The entered shelter date displayed at the top of the record is taken from the date brought in field in the entry slider. If you update the date there, it will update the entered shelter date.

If you have the option active "once assigned codes cannot be changed" in Settings->Options->Animal Codes, you will have to turn this off first if the animal code uses the date brought in.

Kind regards,

Jon
